app/templates/package.json

```diff
@@ -1,8 +1,15 @@
 {
-  "name": "<%= appName %>",
+  "name": "do-not-commit",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "babel-eslint": "^7.1.1",
+    "eslint": "^3.11.1",
+    "eslint-config-zeal": "^0.13.0",
```

Comment:
I released 0.14.1 today which supports the eslint and plugin versions listed here.